mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/CassandraBlobsDAO.java

```diff
@@ -168,7 +168,7 @@ private PreparedStatement prepareInsertPart(Session session) {
 
     private byte[] concatenateDataParts(Stream<BlobPart> blobParts) {
         ImmutableList<byte[]> parts = blobParts
-            .map(blobPart -> OptionalUtils.ifEmpty(
+            .map(blobPart -> OptionalUtils.peekOnEmpty(
```

Comment:
Is it just me or this name is really worst than `ifEmpty`

Reply:
I agree